polaris-for-vscode/scripts/prepublish.js

```diff
@@ -13,8 +13,9 @@ const filePath = path.join(dirPath, outFile);
 const groupedTokens = getGroupedCustomPropertyNames();
 const tokens = getCustomPropertyNames();
 
-// we don't need legacy tokens
+// we don't need legacy or color.dark tokens
 delete groupedTokens['legacy-tokens'];
+delete groupedTokens['color.dark'];
```

Comment:
There's no `groupedTokens` with a `color.dark` key: https://github.com/Shopify/polaris/blob/20ea614e6d47566374d61eb2519008fa77d0e454/polaris-react/scripts/utilities/getCustomPropertyNames.js#L34-L36

You may need to filter that out in the `getGroupedCustomPropertyNames` util e.g.
`fs.readdirSync(tokenGroupsDir).filter(fileName => !fileName.startsWith('color.dark')).map(...)`